src/databricks/labs/ucx/cli.py

```diff
@@ -149,6 +149,47 @@ def revert_migrated_tables(schema: str, table: str, *, delete_managed: bool = Fa
         tm.revert_migrated_tables(schema, table, delete_managed=delete_managed)
 
 
+def migrate_uc_to_uc(
+    from_catalog: str,
+    from_schema: str,
+    from_table: list[str],
+    to_catalog: str,
+    to_schema: str,
+):
+    logger.info("Running migrating uc objects command")
+    ws = WorkspaceClient()
+    installation_manager = InstallationManager(ws)
+    installation = installation_manager.for_user(ws.current_user.me())
+    if not installation:
+        logger.error(CANT_FIND_UCX_MSG)
+        return
+    sql_backend = StatementExecutionBackend(ws, installation.config.warehouse_id)
+    tmp = TableMapping(ws)
+    tables = TablesMigrate(
+        TablesCrawler(backend=sql_backend, schema=installation.config.inventory_database),
+        ws=ws,
+        backend=sql_backend,
+        tm=tmp,
+    )
+    if from_catalog == "" or to_catalog == "":
+        logger.error("Please enter from_catalog and to_catalog details")
+        return
+    if from_schema == "" or to_schema == "" or from_table == "":
+        logger.error("Please enter from_schema, to_schema and from_table(enter * for migrating all tables) details.")
+        return
+    if from_catalog == to_catalog and from_schema == to_schema:
+        logger.error("please select a different schema or catalog to migrate to")
+        return
+    logger.error(f"migrating tables {from_table} from {from_catalog}.{from_schema} to {to_catalog}.{to_schema}")
+    tables.migrate_uc_tables(
```

Comment:
If you want to support wildcard, check for "*" value and iterate over tables from that schema and call this method